doc/contributing/api-documentation.md

```diff
@@ -11,13 +11,21 @@ generate the following:
 1. Human-readable HTML in `out/doc/api/*.html`
 2. A JSON representation in `out/doc/api/*.json`
 
-These are published to nodejs.org for multiple versions of Node.js. As an
+These artifacts published to nodejs.org for multiple versions of Node.js. As an
 example the latest version of the human-readable HTML is published to
 [nodejs.org/en/doc](https://nodejs.org/en/docs/), and the latest version
 of the json documentation is published to
 [nodejs.org/api/all.json](https://nodejs.org/api/all.json)
 
-<!-- TODO: Add docs about how the publishing process happens -->
+The artifacts are built as part of release builds by running the doc-upload
+Makefile target as part of the release-sources part of the
+[iojs+release job](https://github.com/nodejs/node/blob/1a83ad6a693f851199608ae957ac5d4f76871485/Makefile#L1218-L1224).
+This target runs the doc target to build the docs and then uses
+scp to copy them onto the staging/www server into a directory of the form
```

Comment:
Do we need to go that deep into the specifics here? Isn't that visible by reading the Makefile?

Reply:
I think the few lines are worth avoiding people having to click into the link.